indico/modules/events/layout/forms.py

```diff
@@ -81,3 +98,16 @@ class MenuPageForm(MenuUserEntry):
 
 class AddImagesForm(IndicoForm):
     image = JSONField("Image", widget=DropzoneWidget(accepted_file_types='image/*'))
+
+
+class CSSSelectionForm(IndicoForm):
+    theme = SelectField(_("Theme"), [Optional()], coerce=lambda x: x)
+
+    def __init__(self, *args, **kwargs):
+        event = kwargs.pop('event')
+        super(CSSSelectionForm, self).__init__(*args, **kwargs)
+        css_file = StylesheetFile.find_first(event_id=event.id)
+        self.theme.choices = list(THEMES)
+        if css_file:
+            self.theme.choices = (THEMES +
+                                  [('_custom', _("Custom CSS file ({file_name})".format(file_name=css_file.filename)))])
```

Comment:
filename is in the dictionary, file_name isn't. <1M results vs almost 200M results on google